src/css_tagger.coffee

```diff
@@ -41,12 +39,26 @@ define ->
       for feature in featureNames
         add feature
 
-
     # Activated permanently
     storage = window.localStorage or { setItem: -> }
     for feature in (JSON.parse(storage.getItem('pinball_wizard')) or [])
       add feature
 
-    ele.className += ' ' + classNames.join(' ') if ele
+    # Activated by the queue
+    for entry in pinballQueue
+      continue unless entry.length
+
+      switch entry[0]
+        when 'activate'
+          add entry[1]
+
+        when 'add'
+          for feature, state of entry[1]
+            if state == 'active'
+              add feature
+            else if state == 'inactive'
```

Comment:
What do you think about a generic `else`? That way it would add disabled features.